sdk/appconfiguration/azure-appconfiguration/azure/appconfiguration/aio/_azure_appconfiguration_client_async.py

```diff
@@ -91,6 +93,7 @@ def __init__(self, base_url: str, credential: AsyncTokenCredential, **kwargs) ->
             authentication_policy=credential_policy,
             user_agent_policy=user_agent_policy,
             per_call_policies=self._sync_token_policy,
+            api_version="1.0",
```

Comment:
Do we allow users to specify api_version?

Reply:
Yes, "api_version" is a keyword. Now changed to set the default value to "1.0".

Reply:
Have reverted the updates.